Objects/dictobject.c

```diff
@@ -1227,6 +1264,101 @@ insert_into_dictkeys(PyDictKeysObject *keys, PyObject *name)
     return ix;
 }
 
+
+static inline int
+insert_unsplit_dict(PyInterpreterState *interp, PyDictObject *mp,
```

Comment:
We call dict doesn't have split values as "combined".
Please use it unless there is special reason to use special word.